templates/pwm.asg.element.json

```diff
@@ -0,0 +1,925 @@
+                                        "",
+                                        [
+                                            "aws s3 cp s3://",
+                                            {
+                                                "Ref": "ConfigBucketName"
+                                            },
+                                            "/pwmcloudwatchagent.json /opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json"
+                                        ]
+                                    ]
+                                }
+                            },
+                            "71-chmod-conf": {
+                                "command": "chmod 640 /opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json"
+                            },
+                            "72-chown-conf": {
+                                "command": "chown root:root /opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json"
+                            },
+                            "80-inject-custom-agent-config": {
+                                "command": {
+                                    "Fn::Join": [
+                                        "",
+                                        [
+                                            "/opt/aws/amazon-cloudwatch-agent/bin/",
+                                            "amazon-cloudwatch-agent-ctl",
+                                            " -a fetch-config -m ec2 -c",
+                                            " file:/opt/aws/amazon-cloudwatch-agent/etc/",
+                                            "amazon-cloudwatch-agent.json -s"
+                                        ]
+                                    ]
+                                }
+                            }
+                        },
+                        "packages": {
+                            "yum": {
+                                "unzip": []
+                            }
+                        },
+                        "files": {
+                            "/opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json": {
+                                "content": {
+                                    "Fn::Join": [
+                                        "",
+                                        [
+                                            "{",
+                                            "\"logs\": {\n",
+                                            "  \"logs_collected\": {\n",
+                                            "    \"files\": {\n",
+                                            "      \"collect_list\": [\n",
+                                            "        {\n",
+                                            "          \"file_path\": \"/usr/share/tomcat/webapps/ROOT/WEB-INF/logs/PWM.log\",\n",
+                                            "          \"log_group_name\": \"PWM18LogGroup\",\n",
```

Comment:
I'd recommend using a log_group_name throughout of `/aws/ec2/lx/${AWS::StackName}`, rather than hardcoding it